main_test.go

```diff
@@ -440,6 +442,38 @@ func writeOCIConfigFile(spec oci.CompatOCISpec, configPath string) error {
 	return ioutil.WriteFile(configPath, bytes, testFileMode)
 }
 
+func newSingleContainerPodStatusList(podID, containerID string, podState, containerState vc.State) []vc.PodStatus {
```

Comment:
this cannot be generic, this should be part of each test, hence cannot be in main_test

Reply:
Why ?

Reply:
different tests different needs

https://github.com/clearcontainers/runtime/blob/master/list_test.go#L607
https://github.com/clearcontainers/runtime/blob/master/run_test.go#L255

Reply:
Ok but the name of the function is specific enough so that it won't collide with other functions, right ? And I need it because I am reusing it across several tests.
We can move this later if we __really__ run into an issue but I cannot see any issue right now.